assets/dev/js/editor/document/hooks/data/document/elements/helper.js

```diff
@@ -46,4 +46,97 @@ export default class Helper {
 			containers[ containers.length - 1 ].model.trigger( 'request:edit' );
 		}
 	}
+
```

Comment:
If functions used only from one 'hook' better to move the function to the hook.

Reply:
@StyleShit status?

Reply:
Done.